snippets/components/video-render.js

```diff
@@ -0,0 +1,147 @@
+class LazyVideoComponent extends HTMLElement {
```

Comment:
Not specific to this file, but the folder itself, make the `/components` at the root and not inside of `/snippets`, thanks.